packages/runtime/src/schema.ts

```diff
@@ -178,23 +178,12 @@ const UpdateOptionsInput = new GraphQLInputObjectType({
   },
 })
 
-const CeramicSignerInputType = new GraphQLInputObjectType({
-  name: 'CeramicSignerInput',
-  fields: {
-    isAuthenticated: { type: GraphQLBoolean },
-  },
-})
-
 const HideOptionsInput = new GraphQLInputObjectType({
   name: 'HideOptionsInput',
   fields: {
-    asDID: {
-      type: GraphQLDID,
-      description: 'DID for the signer(Optional)',
-    },
-    signer: {
-      type: CeramicSignerInputType,
-      description: 'Ceramic signer (Optional)',
+    shouldIndex: {
```

Comment:
Could you use https://github.com/ceramicstudio/js-composedb/blob/8cdf4ad26012cc10cf12d757c801d8000d35558f/packages/runtime/src/schema.ts#L152 directly so we have a single reference for this field please?